packages/asc-ui/src/components/CardContainer/CardContainer.stories.tsx

```diff
@@ -0,0 +1,287 @@
+        </CardActions>
+      </Card>
+      <Card horizontal>
+        <CardMedia backgroundColor="level2">
+          <Icon size={25}>
+            <Download />
+          </Icon>
+        </CardMedia>
+        <CardContentStyled>
+          <Heading as="h6">This is a card</Heading>
+          <Paragraph>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Architecto, quisquam!
+          </Paragraph>
+        </CardContentStyled>
+        <CardActions>
+          <Icon size={15}>
+            <ChevronRight />
+          </Icon>
+        </CardActions>
+      </Card>
+    </CardContainer>
+  ))
+  .add('segment "Generieke functies"', () => (
+    <CardContainerGeneriekeFuncties maxWidth={460}>
+      <Card horizontal>
+        <CardContentStyled>
+          <Heading as="h6">This is a card</Heading>
+          <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+        </CardContentStyled>
+        <CardActions>
+          <Icon size={15}>
+            <ChevronRight />
+          </Icon>
+        </CardActions>
+      </Card>
+      <Card horizontal>
+        <CardContentStyled>
+          <Heading as="h6">This is a card</Heading>
+          <Paragraph>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Architecto, quisquam!
+          </Paragraph>
+        </CardContentStyled>
+        <CardActions>
+          <Icon size={15}>
+            <ChevronRight />
+          </Icon>
+        </CardActions>
+      </Card>
+      <Card horizontal>
+        <CardContentStyled>
+          <Heading as="h6">This is a card</Heading>
+          <Paragraph>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Explicabo,
+            provident quasi.
+          </Paragraph>
+        </CardContentStyled>
+        <CardActions>
+          <Icon size={15}>
+            <ChevronRight />
+          </Icon>
+        </CardActions>
+      </Card>
+      <Card horizontal>
+        <CardContentStyled>
+          <Heading as="h6">This is a card</Heading>
+          <Paragraph>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Explicabo,
+            provident quasi voluptate fuga nulla. Lorem ipsum dolor sit amet,
+            consectetur adipisicing elit. Explicabo, provident quasi voluptate
+            fuga nulla.
+          </Paragraph>
+        </CardContentStyled>
+        <CardActions>
+          <Icon size={15}>
+            <ChevronRight />
+          </Icon>
+        </CardActions>
+      </Card>
+      <Card horizontal>
+        <CardContentStyled>
+          <Heading as="h6">This is a card</Heading>
+          <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+        </CardContentStyled>
+        <CardActions>
+          <Icon size={15}>
+            <ChevronRight />
+          </Icon>
+        </CardActions>
+      </Card>
+    </CardContainerGeneriekeFuncties>
+  ))
+  .add('segment "Over Data & Over deze site"', () => (
+    <CardContainerOverData>
```

Comment:
Use English naming

Reply:
@royderks `Over Data & Over deze site` -> `About Data & About this site` ?

Reply:
Done